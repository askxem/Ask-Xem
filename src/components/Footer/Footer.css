.footerDiv {
  width: 100vw;
  position: absolute;
  bottom: 0px;
  display: flex;
  justify-content: center;
  padding-bottom: 1%;
}
