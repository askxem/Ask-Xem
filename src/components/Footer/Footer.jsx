import styles from './Footer.css'
import { Link } from 'react-router-dom'

export default function Footer() {
  return (
    <div className={styles.footerDiv}>
      <Link to={'/about'}>meet the Ask Xem team</Link>
    </div>
  )
}
