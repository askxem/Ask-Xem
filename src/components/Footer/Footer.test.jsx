import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import Footer from './Footer.jsx'

it('renders footer component', () => {
  const { container } = render(
    <MemoryRouter>
      <Footer />
    </MemoryRouter>
  )

  expect(container).toMatchSnapshot()
})
