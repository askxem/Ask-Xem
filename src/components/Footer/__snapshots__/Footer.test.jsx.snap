// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer component 1`] = `
<div>
  <div
    class="footerDiv"
  >
    <a
      href="/about"
    >
      meet the Ask Xem team
    </a>
  </div>
</div>
`;
